omnipaxos/src/omni_paxos.rs

```diff
@@ -393,14 +393,18 @@ where
 
 /// An error indicating a failed proposal due to the current cluster configuration being already stopped
 /// or due to an invalid proposed configuration. Returns the failed proposal.
-#[allow(missing_docs)]
 #[derive(Debug)]
 pub enum ProposeErr<T>
 where
     T: Entry,
 {
+    /// Couldn't propose entry because cluster is stopped. Contains failed, proposed entry.
     PendingReconfigEntry(T),
+    /// Couldn't propose reconfiguration because cluster is already stopped. Contains failed, proposed
```

Comment:
```suggestion
    /// Couldn't propose reconfiguration because a reconfiguration is already pending. Returns the failed, proposed `ClusterConfig` and the metadata.
```